pkg/scheduler/plugins/proportion/capacity_policy/capacity_policy_test.go

```diff
@@ -154,7 +154,7 @@ var _ = Describe("Capacity Policy Check", func() {
 				testName := name
 				testData := data
 				It(testName, func() {
-					capacityPolicy := New(testData.queues, true)
```

Comment:
Can you maybe remove the parameter from here?
https://github.com/NVIDIA/KAI-Scheduler/blob/2145d068b70ee5c88b533b98635b5cc74cf96ff4/pkg/scheduler/plugins/proportion/capacity_policy/capacity_policy.go#L20

After your changes in `max_allowed_check.go` and `quota_check.go` it is no longer needed and it will probably streamline the changes in `capacity_policy_test.go`.